tests/e2e/conftest.py

```
@@ -0,0 +1,291 @@
+
+    cert = _x509.load_pem_x509_certificate(cert_path.read_bytes())
+    return cert.fingerprint(_h.SHA1()).hex().upper()
+
+
+def _trust_cert_in_root_store(cert_path: Path) -> str:
+    """
+    Import cert into CurrentUser\\Root (Schannel trusts both LocalMachine and
+    CurrentUser root stores; the user store requires no administrator rights).
+    Returns the SHA-1 thumbprint for later removal.
+    """
+    result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)